src/site/content/en/blog/native-file-system/index.md

```diff
@@ -246,27 +286,40 @@ butDir.addEventListener('click', async (e) => {
 });
 ```
 
+## What's new/changed? {: #whats-new }
+
+Chrome 82
+
+* Support for [writable streams](#save-to-disk) was added, and the previous
+  method for writing to disk (`FileSystemWriter`) was deprecated.
```

Comment:
not sure if "deprecated" is the right word here, we actually removed the previous method.

Reply:
Changed to removed.